fiftyone/utils/sam3.py

```
@@ -0,0 +1,610 @@
+            if score < self._confidence_threshold:
+                continue
+
+            x_norm, y_norm, w_norm, h_norm = box_xywh
+            bounding_box = [
+                max(0.0, float(x_norm)),
+                max(0.0, float(y_norm)),
+                min(1.0 - x_norm, float(w_norm)),
+                min(1.0 - y_norm, float(h_norm)),
+            ]
+
+            x1_int = max(0, int(round(x_norm * frame_width)))
```

Ruff (0.14.14)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)